scss/components/time.scss

```diff
@@ -11,17 +11,17 @@ $block: #{$fd-namespace}-time;
       max-width: fd-space(10);
       text-align: center;
       margin-right: fd-space(1);
+
```

Comment:
I think I missed this before (probably because it didn't show up directly in the file diff), but what exactly is this `width: 184px` for?  The phrase "quick fix" worries me.  And _what_ overflow issue?  This _seems_ like it might be related to the use of `fd-time` within `fd-time-picker`, but this strikes me as a problem. In fact, see the attached screenshot.

![Screen Shot 2019-08-19 at 9 49 30 AM](https://user-images.githubusercontent.com/11407353/63275334-fb4b9c80-c266-11e9-99bd-b53771422103.png)

The 4th `__item` is wrapping due to this arbitrary width.  I know it's the playground with crazy margin values being applied, but there is still no reason why any of this should wrap to a new line.

Maybe the `fd-time` class should provide a flex box wrapper around the `fd-time__item` blocks to prevent them from ever "wrapping".  Thoughts?

Reply:
I agree that it should be changed (any fixed width seems like a hack), but I wasn't sure this was the right time to do it...

Reply:
You know what, maybe it isn't the time to do it.  I've created #280 to track this work.